.travis.yml

```diff
@@ -17,13 +26,22 @@ addons:
     - build-essential
     - libacl1-dev
     - libsystemd-journal-dev
+    - gcc-aarch64-linux-gnu
+    - libc6-dev-arm64-cross
 
 install:
   -
 
 script:
- - ./autogen.sh
  # Build host and fly to ensure we build stage1 init. We don't build everything
  # to avoid downloading large images.
- - ./configure --with-stage1-flavors=host,fly --enable-tpm=auto
- - make unit-check
+ -      if [ "${TARGET}" == "amd64" ]; then
+                ./autogen.sh;
+                ./configure --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make unit-check;
+        elif [ "${TARGET}" == "arm64" ]; then
+                eval "$(GIMME_ARCH=${TARGET} GIMME_CGO_ENABLED=1 ./gimme.local ${TRAVIS_GO_VERSION})";
+                ./autogen.sh;
+                ./configure --host=aarch64-linux-gnu --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make V=2;
```

Comment:
I get this when I try: ```go test: -race is only supported on linux/amd64, freebsd/amd64, darwin/amd64 and windows/amd64```.

I plan to setup our local Jenkins to do tests on an arm64 platform.  Do you have any other suggestions?